### ISSUE TITLE
Enable Cider use outside of Flutter packages

### ISSUE THREAD

**Issue by LPLabrys99** [f3ath/cider#61]
I just started using Cider for changelog management within a mono-repo. It's working pretty well for Flutter modules.

I would also like to use Cider to manage other changelog files which would belong to other parts of the repository like the Firebase backend and CI/CD pipelines. Unfortunately Cider expects there to be a `pubspec.yaml` file present and fails if there isn't one in the current repo.

Would it be practical to ease the requirements for a pubspec.yaml file for certain actions like adding to the changelog and creating new semantic versions within the changelog?

Thanks

**Comment by f3ath** [comment]
Makes sense to me. There's no reason to require pubspec for non-mutating operations on the changelog.
